README.md

```diff
@@ -1,70 +1,18 @@
-# Getting Started with Create React App
+# Project Three: Roller Derby Wheel Selector
```

Comment:
Somehow include this readme info on the live site so users know the purpose of the site - if not through routing, a simple About button with state and ternary could work to display/hide info on button click